src/libraries/System.Net.Quic/src/System/Net/Quic/Implementations/MsQuic/Interop/MsQuicException.cs

```diff
@@ -8,12 +8,12 @@
 
 namespace Microsoft.Quic
 {
-    internal sealed class MsQuicException : QuicException
+    internal sealed class MsQuicException : Exception
```

Comment:
I think we can get rid of it if we enhance the `QuicException` message with the MsQuic status string representation. That's the only interesting thing this brings and I think it might be valuable for diagnostics. E.g. getting somewhere in the message `QUIC_STATUS_INVALID_STATE` rather than just `0x8007139F` as an `HResult`.

Reply:
I agree. I never liked Internal Exceptions. We can do pretty print got the status as well as set Result with raw platform specific error.

Reply:
Removed.

Btw. now we don't really need the `innerException` parameter on `QuicException`, So we can remove it if we want to simplify the API a bit.